docs/src/rpc/index.md

```diff
@@ -29,30 +31,79 @@ Those methods instead accept and produce account RPC handles, which are random s
 For example, `/stdlib/newTestAccount` does not return an account like `{!js} newTestAccount`, but instead returns an account RPC handle.
 
 + `/forget/acc` accepts an account RPC handle and deletes it from the Reach RPC Server's memory.
-+ `/acc/$METHOD` where `$METHOD` is a method of an account representation of the [JavaScript standard library](##ref-frontends-js).
+
+### {#ref-rpc-method-acc} Account Methods
 
 All `/acc` methods are synchronous value RPC methods that accept and produce the same arguments and return values as the corresponding function, encoded as JSON objects, except they accept an additional first argument, which is the account RPC handle returned by a prior RPC method invocation; and, a method that accepts a backend (like `/acc/attach` (i.e. `{!js} acc.attach`) or `/acc/deploy` (i.e. `{!js} acc.deploy`) does not accept a backend argument, but has it implicitly provided by the Reach RPC Server.
 
++ `/acc/$METHOD` where `$METHOD` is a method of an account representation of the [JavaScript standard library](##ref-frontends-js).
+
 Furthermore, those that produce contract representations instead produce contract RPC handles.
 For example, `/acc/deploy` does not return a contract representation like `{!js} acc.deploy`, but instead returns a contract RPC handle.
 
 + `/forget/ctc` accepts a contract RPC handle and deletes it from the Reach RPC Server's memory.
-+ `/ctc/$METHOD` where `$METHOD` is a method of a contract representation of the [JavaScript standard library](##ref-frontends-js).
+
+### {#ref-rpc-method-ctc} Contract Methods
 
 Most `/ctc` methods are synchronous value RPC methods that accept and produce the same arguments and return values as the corresponding function, encoded as JSON objects, except they accept an additional first argument, which is the contract RPC handle returned by a prior RPC method invocation.
 
++ `/ctc/$METHOD` where `$METHOD` is a method of a contract representation of the [JavaScript standard library](##ref-frontends-js).
+
+`/ctc` methods can access `{!js} ctc.views`, `{!js} ctc.apis`, and `{!js} ctc.events`. 
+Similar to standard library functions, the full name or first letter can be used when calling the following methods.
+
++ `/ctc/v` or `/ctc/views` provides access to all `{!rsh} View`s.
++ `/ctc/a` or `/ctc/apis` makes `{!rsh} API` functions available to the frontend.
++ `/ctc/e` or `/ctc/events` provides access to `{!rsh} Events`.
+
+Any `{!rsh} View`, `{!rsh} API` or `{!rsh} Events` may be accessed with the following syntax:
+
+```
+RPC /ctc/views/$VIEW_NAME            # Top-level data of a view, $VIEW_NAME
```

Comment:
I don't really understand the comment here.

Reply:
I've simplified comments and added examples for APIs and Events